### PR TITLE
Update for name change Rabo OmniKassa to Smart Pay

### PR DESCRIPTION
**Pull request by rvdsteege**
For https://github.com/pronamic/wp-pronamic-pay-omnikassa-2/issues/13

### CODE REVIEW

**Comment by remcotolsma** [comment]
1. `Rabobank - Rabo Smart Pay`
2. `Rabobank - Smart Pay`

I think option 1 `Rabobank - Rabo Smart Pay` is a bit more complete.

```
{provider_name} - {gateway_name}
```

Schermafbeelding 2022-10-18 om 12 45 39

File: pronamic-ideal.php [hunk 1]
```diff
@@ -517,11 +517,11 @@ function( $gateways ) {
 			]
 		);
 
-		// Rabobank - OmniKassa 2.0.
+		// Rabobank - Smart Pay.
```
Comment (remcotolsma) [inline pronamic-ideal.php:520]: Rabobank - Rabo Smart Pay

File: pronamic-ideal.php [hunk 2]
```diff
@@ -517,11 +517,11 @@ function( $gateways ) {
...
 		$gateways[] = new \Pronamic\WordPress\Pay\Gateways\OmniKassa2\Integration(
 			[
 				'id'      => 'rabobank-omnikassa-2',
-				'name'    => 'Rabobank - OmniKassa 2.0',
+				'name'    => 'Rabobank - Smart Pay',
```
Comment (remcotolsma) [inline pronamic-ideal.php:524]: Rabobank - Rabo Smart Pay

File: pronamic-ideal.php (hunk 1 above)
Comment (remcotolsma) [inline pronamic-ideal.php:520]: Rabobank - Rabo Smart Pay?
https://github.com/pronamic/wp-pronamic-pay/pull/344#issuecomment-1282192927

File: pronamic-ideal.php (hunk 2 above)
Comment (remcotolsma) [inline pronamic-ideal.php:524]: Rabobank - Rabo Smart Pay?
https://github.com/pronamic/wp-pronamic-pay/pull/344#issuecomment-1282192927

File: pronamic-ideal.php [hunk 3]
```diff
@@ -530,7 +530,7 @@ function( $gateways ) {
 		$gateways[] = new \Pronamic\WordPress\Pay\Gateways\OmniKassa2\Integration(
 			[
 				'id'      => 'rabobank-omnikassa-2-sandbox',
-				'name'    => 'Rabobank - OmniKassa 2.0 - Sandbox',
+				'name'    => 'Rabobank - Smart Pay - Sandbox',
```
Comment (remcotolsma) [inline pronamic-ideal.php:533]: Rabobank - Rabo Smart Pay?
https://github.com/pronamic/wp-pronamic-pay/pull/344#issuecomment-1282192927

**Review by remcotolsma — COMMENTED** [review]
Rabobank - Rabo Smart Pay?
https://github.com/pronamic/wp-pronamic-pay/pull/344#issuecomment-1282192927

**Comment by rvdsteege** [comment]
They also seem to use Rabo OnlineKassa specifically for online payments / webshops. Would that be even more appropriate?

- https://www.rabobank.nl/bedrijven/betalen/klanten-laten-betalen/rabo-smart-pay/rabo-onlinekassa

**Pull request merged**